static/app/views/settings/project/projectReplays.tsx

```diff
@@ -30,7 +30,7 @@ function ProjectReplaySettings({organization, project, params: {projectId}}: Pro
             external
             href="https://docs.sentry.io/product/session-replay/replay-page-and-filters/"
           >
-            {t('Read the docs')}
+            {t('Read the Docs')}
```

Comment:
extra change because this was inconsistent with other header actions in the settings, which all have the first word capitalized